### PR TITLE
Fixing UI issues

### PR DESCRIPTION
**Pull request by kavishkafernando**
This PR fixes the following issues,

- Issue https://github.com/wso2/product-apim/issues/5754
- Remove tooltips in overview stepper component when requirements are completed.

- Arrow overlapping issue when searching for component. (See below)

Screen Shot 2019-10-14 at 9 58 03 AM

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/NewOverview/CustomizedStepper.jsx [hunk 1]
```diff
@@ -314,7 +314,10 @@ export default function CustomizedSteppers() {
                 <Step className={classes.label}>
                     <StepLabel style={{ position: 'relative' }} >
                         <Box p={2} bgcolor='white' borderLeft='0' borderRight='0' >
-                            <Tooltip title='You have to specify an Endpoint for the API' placement='top'>
+                            <Tooltip
+                                title={isEndpointAvailable ? '' : 'You have to specify an Endpoint for the API'}
```
Comment (Rajith90) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/NewOverview/CustomizedStepper.jsx]:
```suggestion
                                title={isEndpointAvailable ? '' : 'You have to specify an endpoint for the API'}
```

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/NewOverview/CustomizedStepper.jsx [hunk 2]
```diff
@@ -332,7 +335,10 @@ export default function CustomizedSteppers() {
                                     </Link>
                                 </Grid>
                             </Tooltip>
-                            <Tooltip title='You have to select the Business plan for the API' placement='bottom'>
+                            <Tooltip
+                                title={isTierAvailable ? '' : 'You have to select the Business plan for the API'}
```
Comment (Rajith90) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/NewOverview/CustomizedStepper.jsx]:
```suggestion
                                title={isTierAvailable ? '' : 'You have to select the business plan for the API'}
```

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/NewOverview/CustomizedStepper.jsx (hunk 1 above)
Comment (kavishkafernando) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/NewOverview/CustomizedStepper.jsx]: Fixed

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/NewOverview/CustomizedStepper.jsx (hunk 2 above)
Comment (kavishkafernando) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/NewOverview/CustomizedStepper.jsx]: Fixed

**Pull request merged**